ai_models/event_detector/event.py

```diff
@@ -5,10 +5,11 @@
 from sklearn.metrics import f1_score, confusion_matrix
 from sklearn.ensemble import RandomForestClassifier
 import joblib
+import compile_data
 
 # Gets the data with which to build and/or test a new model
 def get_data():
-    data = pd.read_fwf(os.path.join('ai_models/event_detector/dataset', 'dataset.txt'))
```

Comment:
Maybe we should just pass in the path as a parameter so we can run tests and stuff without breaking run.py, which uses `ai_models/event_detector/dataset`